transmart-core-db/grails-app/services/org/transmartproject/db/log/AccessLogService.groovy

```diff
@@ -3,14 +3,25 @@
 package org.transmartproject.db.log
 
 import grails.transaction.Transactional
+import groovy.json.JsonBuilder
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
 import org.transmartproject.core.log.AccessLogEntryResource
 import org.transmartproject.core.users.User
 
 @Transactional
 class AccessLogService implements AccessLogEntryResource {
 
+    static final Logger log = LoggerFactory.getLogger(AccessLogService.class)
+
     @Override
     AccessLogEntry report(Map<String, Object> additionalParams = [:], User user, String event) {
```

Comment:
These logs will also end up in the database (and if transmartApp is available, in the admin panel there). Seems like these things should not be mixed.

Reply:
But the requirement is to add them both to the database and (if configured) to the log file. You mean it should be a separate function or I should use a separate services?